coarse/models/errors.py

```
@@ -0,0 +1,13 @@
+from enum import Enum
+from typing import List
+from pydantic import BaseModel
+from coarse.models.sections import SectionLabel
+
+class ErrorEntry(BaseModel):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)